driver/bpf/filler_helpers.h

```diff
@@ -19,6 +19,9 @@ or GPL2.txt for full copies of the license.
 
 #include "../ppm_flag_helpers.h"
 
+// Maximum var size allowed by the verifier
+#define BPF_MAX_VAR_SIZE 1ULL << 29
```

Comment:
Is this value documented anywhere? Or, even better, defined in a kernel header that we can pull in?

Reply:
Yeah, it's in [bpf_verifier.h](https://github.com/torvalds/linux/blob/master/include/linux/bpf_verifier.h#L20).

We decided to define it here also to have a more complete (last `E`) name

Reply:
Anyways, that value is now used [here in the verifier](https://elixir.bootlin.com/linux/v5.6.3/source/kernel/bpf/verifier.c#L3599).

Reply:
We need the kernel headers (for a particular kernel version) anyway, right? If we cannot simply `#include <linux/bpf_verifier.h>`, can we at least add a comment referring to it?

Reply:
Agree.

Added a comment line referring to the source of `BPF_MAX_VAR_SIZE` (ie., `BPF_MAX_VAR_SIZ`).